var/spack/repos/builtin/packages/py-cryptography/package.py

```diff
@@ -32,7 +32,7 @@ class PyCryptography(PythonPackage):
     depends_on("py-setuptools@40.6:", when="@2.7:36", type="build")
     depends_on("py-setuptools@18.5:", when="@2.2:2.6", type="build")
     depends_on("py-setuptools@11.3:", when="@:2.1", type="build")
-    depends_on("py-setuptools-rust@0.11.4:", when="@3.4:", type=("build", "run"))
+    depends_on("py-setuptools-rust@0.11.4:", when="@3.4:", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools-rust@0.11.4:", when="@3.4.2:", type="build")
```
Just because we actually know when this requirement first appears. I also wouldn't be opposed to adding the build/run dep for 3.4.0:3.4.1 but can understand why you wouldn't want to.